src/libs/sfdk/vboxvirtualmachine.cpp

```diff
@@ -847,12 +847,11 @@ bool VBoxVirtualMachinePrivate::isVirtualMachineRunningFromInfo(const QString &v
 QStringList VBoxVirtualMachinePrivate::listedVirtualMachines(const QString &output)
 {
     QStringList vms;
-    QRegExp rx(QLatin1String("\"(.*)\""));
-    rx.setMinimal(true);
-    int pos = 0;
-    while ((pos = rx.indexIn(output, pos)) != -1) {
-        pos += rx.matchedLength();
-        vms.append(rx.cap(1));
+    QRegularExpression re(QLatin1String("\"(?!<inaccessible>)(.*)\""));
```

Comment:
That's what I though first about doing too, but then I found it less optimal. It's hardly readable. IIUC your regexp is also not 100% correct, but that's not critical here. How about to rather keep the regexp general and do the filtering under the while loop for better readability?

Reply:
I believe that any regexp is easier to throw away and write a new one, rather than modify it. So I agree. Updated PR.

Reply:
yeah, one of those write-only languages :)